.github/workflows/frontend.yml

```diff
@@ -3,9 +3,6 @@ name: frontend
 on:
   pull_request:
   merge_group:
-    paths:
-      - '**.js'
-      - '**package.json'
```

Comment:
Same as a above, would it make sense to filter paths on `pull_request` but not `merge_group`?